app/models/server/raw/index.ts

```diff
@@ -74,3 +77,4 @@ export const CustomUserStatus = new CustomUserStatusRaw(CustomUserStatusModel.mo
 export const LivechatAgentActivity = new LivechatAgentActivityRaw(LivechatAgentActivityModel.model.rawCollection());
 export const Statistics = new StatisticsRaw(StatisticsModel.model.rawCollection());
 export const NotificationQueue = new NotificationQueueRaw(NotificationQueueModel.model.rawCollection());
+export const ServerEvents: IServerEventsRepository = new ServerEventsRaw(ServerEventModel.model.rawCollection());
```

Comment:
`ServerEvents` actually has the type `ServerEventsRaw` right? you should not cast the interface type to it